modules/visual_script/visual_script_nodes.h

```diff
@@ -103,6 +103,103 @@ class VisualScriptFunction : public VisualScriptNode {
 	VisualScriptFunction();
 };
 
+class VisualScriptLists : public VisualScriptNode {
```

Comment:
I was talking about this @fire I am thinking about improving the name...

Reply:
Did you think of any ideas?

Reply:
I think Lists is fine for now and considering it's a VirtualClass I don't think it will a breaking change even if we try to fix it later.